common/web/lm-worker/build.sh

```diff
@@ -97,7 +97,9 @@ fi
 
 if builder_start_action clean || builder_has_action build; then
   npm run clean
-  builder_finish_action success clean
+  if builder_has_action clean; then
+    builder_finish_action success clean
+  fi
```

Comment:
![image](https://user-images.githubusercontent.com/25213402/191646450-b62f88b0-937c-4353-b809-11505cd34665.png)

Fixes the warning in yellow shown above.